config/locales/devise.sv.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to sv.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+sv:
+  devise:
+    confirmations:
+      confirmed: "Din e-postadress har bekräftats."
+      send_instructions: "Du kommer att få ett mail med instruktioner för hur du bekräftar din e-postadress om några minuter."
+      send_paranoid_instructions: "Om din e-postadress finns i vår databas får du ett mail med instruktioner för hur du bekräftar din e-postadress om några minuter."
+    failure:
+      already_authenticated: "Du är redan inloggad."
+      inactive: "Ditt konto är inte aktiverat än."
+      invalid: "Ogiltigt %{authentication_keys} eller lösenord."
+      locked: "Ditt konto är låst."
+      last_attempt: "Du har ytterligare ett försök innan ditt konto är låst."
+      not_found_in_database: "Ogiltigt %{authentication_keys} eller lösenord."
+      timeout: "Din session löpte ut. Vänligen logga in igen för att fortsätta."
+      unauthenticated: "Du måste logga in eller registrera dig innan du fortsätter."
+      unconfirmed: "Du måste bekräfta din e-postadress innan du fortsätter."
+    mailer:
+      confirmation_instructions:
+        subject: "Bekräftelsesinstruktioner"
+      reset_password_instructions:
+        subject: "Återställ lösenordsinstruktioner"
```

Comment:
Should be: "Instruktioner för lösenordsåterställning"

Right now is literally: "Reset the password-instructions" 🙂